msg/en_US/utils.msg

```diff
@@ -241,9 +241,6 @@ usage: %2$s heartbeat <command> [args]\n\
 Available command:\n\
     start          [database-name]\n\
     stop           [-h <host-name>] [-i] [database-name]\n\
-    copylogdb      <start|stop> [-h <host-name>] database-name node-name\n\
```

Comment:
I've found that other msgs are about these utilities with text search in the msg file. Don't they need to be removed?
1. cub_admin 
2. utility help msg for itself: copylogdb, applylogdb. Are they left since any path can't print them?

and applyinfo will be handled in another issue?